snapcraft/plugins/v2/cmake.py

```diff
@@ -69,7 +70,12 @@ def _get_cmake_configure_command(self) -> str:
             c.startswith("-DCMAKE_INSTALL_PREFIX=")
             for c in self.options.cmake_parameters
         ):
-            cmd.append('-DCMAKE_INSTALL_PREFIX="${SNAPCRAFT_CMAKE_INSTALL_PREFIX}"')
+            cmd.append("-DCMAKE_INSTALL_PREFIX=/")
+
+        if not any(
+            c.startswith("-DCMAKE_BUILD_TYPE=") for c in self.options.cmake_parameters
+        ):
+            cmd.append("-DCMAKE_BUILD_TYPE=Release")
```

Comment:
Last Q, do we really need to even specify this at all?  Perhaps just leave it up to the project?

Reply:
ok, will remove, but this is the point of no return.